lib/koans/07_pattern_matching.ex

```diff
@@ -8,12 +8,14 @@ defmodule PatternMatching do
   koan "A pattern can change" do
     a = 1
     a = 2
-    assert (a == 2) == ___
+    assert a == ___
   end
 
   koan "A pattern can also be strict" do
     a = 1
-    assert ^a = ___
+    assert_raise ___, fn() ->
```

Comment:
I think figuring out that a `MatchError` was raised is a little too hard.
How about we add `assert raise_happend(fn() -> ... end)`?
`raise_happend` would just return true if something raised, false otherwise...